src/engine/renderer/tr_image.cpp

```diff
@@ -1978,22 +2013,25 @@ image_t        *R_FindCubeImage( const char *imageName, int bits, filterType_t f
 		}
 	}
 
-	// try to load .CRN cubemap
-	LoadCRN( buffer, pic, &width, &height, &numLayers, &numMips, &bits, 0 );
-	if( numLayers == 6 && pic[0] ) {
-		numPicsToFree = 1;
-		goto createCubeImage;
-	} else {
-		R_FreeCubePics( pic, numLayers );
-	}
+	const char *cubeMapName;
+	char cubeMapBaseName[ MAX_QPATH ];
 
-	// try to load .KTX cubemap
-	LoadKTX( buffer, pic, &width, &height, &numLayers, &numMips, &bits, 0 );
-	if( numLayers == 6 && pic[0] ) {
-		numPicsToFree = 1;
-		goto createCubeImage;
-	} else {
-		R_FreeCubePics( pic, numLayers );
+	COM_StripExtension3( buffer, cubeMapBaseName, MAX_QPATH );
```

Comment:
It's better to use `sizeof(cubeMapBaseName)` since that's more resistant to the buffer size being changed later.